build/common/worker/docker-entrypoint.sh

```diff
@@ -46,8 +46,8 @@ function configureEnv() {
 }
 
 function checkConnection() {
-  su frappe -c ". /home/frappe/frappe-bench/env/bin/activate \
```

Comment:
this privilege escalation with `su` forces all containers to be run with a very dangerous permission to escalate to root